applications/app/templates/serviceWorker.scala.js

```diff
@@ -92,63 +92,3 @@ this.addEventListener('fetch', function (event) {
         event.respondWith(forbidden);
     }
 });
-
-self.addEventListener('push', function (event) {
```

Comment:
I spotted it the cache usage, but not sure I understand why we need it. we don't explicitly push, so this appears to behave like traditional caching? Thought that the cache will still  be consulted anyway. If it isn't accessed by default, it's odd for the cache spec to implicitly `push`, but not implicitly `match`.